src/maintainer/knowledge_base.rst

```diff
@@ -1536,3 +1536,37 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Perl package
+==================
+
+The most commonly used build system for perl packages is ``ExtUtils::MakeMaker``. 
+For the archetypical packaging of packages of this form, you can have a look at ``perl-file-which``'s recipe.
```

Comment:
Super small comment: Maybe add a link to the package here?

Reply:
> Super small comment: Maybe add a link to the package here?

Sure sir! Will do that as well

Reply:
@BastianZim  from where can I get the package link?😅

Reply:
https://github.com/conda-forge/perl-file-which-feedstock

All of conda-forge‘s feedstocks are listed at https://conda-forge.org/feedstock-outputs/

Reply:
I would actually link to the meta.yaml directly:

https://github.com/conda-forge/perl-file-which-feedstock/blob/main/recipe/meta.yaml